app/src/main/java/com/yagni/model/MarketRepo.java

```diff
@@ -7,146 +7,158 @@
 
 public class MarketRepo {
 
-    //Best customers by doller amount 
+    // Best customers by dollar amount
     private static String firstCustomer;
     private static String secondCustomer;
     private static String thirdCustomer;
 
-    //Most ordered product
+    // Most ordered product and their dollar amount
     private static String firstProduct;
     private static String secondProduct;
     private static String thirdProduct;
     private static String firstAmt;
     private static String secAmt;
     private static String thirdAmt;
 
-  
-     public void dayReport(DbConnection linkDb){
-        try{
+    public void dayReport(DbConnection linkDb) {
+        try {
 
             Statement state = linkDb.getConnection().createStatement();
 
-            //============FOR DAILY REPORT ===========//
-            ResultSet result = state.executeQuery("SELECT product_id, sum(product_quanity) AS 'sum' FROM yagni_inv_db.purchase_history WHERE order_date > DATE_ADD(curdate(),INTERVAL -1 DAY) AND order_date <= curdate() GROUP BY product_id ORDER BY sum(product_quanity) DESC LIMIT 3;");
+            /******** FOR DAILY REPORT ********/
+            ResultSet result = state.executeQuery(
+                    "SELECT product_id, sum(product_quanity) AS 'sum' FROM yagni_inv_db.purchase_history WHERE order_date > DATE_ADD(curdate(),INTERVAL -1 DAY) AND order_date <= curdate() GROUP BY product_id ORDER BY sum(product_quanity) DESC LIMIT 3;");
             result.next();
-            firstProduct = result.getString(1);      //1st "most popular order"
+            firstProduct = result.getString(1);         // 1st "most popular order"
```

Comment:
Inline code comments are only for variable declarations. Anything that explains functionality should be a comment above the statement/block of code. In this instance you could probably write one line to encompass what all three statements are doing in this case.